media/js/careers/filters.js

```diff
@@ -30,17 +43,18 @@
         /**
          * When a filter changes, refresh the position list.
          */
-        onFilterChange: function() {
+        onFilterChange: function () {
             var i;
             // collection of tr.position elements
-            var positions = this.positionTable.getElementsByClassName('position');
+            var positions =
+                this.positionTable.getElementsByClassName('position');
```

Comment:
Odd for `prettier` to break a line like this? Or are we running a shorter line-length than the 150chars we use for Python? (Fine if so, just wondering...)

Reply:
Line length in prettier is a rough limit of ~80 characters for readability. It is one of the few configurable options, but I'm not going down that path here 😉 

https://prettier.io/docs/en/options.html#print-width